lib/web/mage/adminhtml/tools.js

```diff
@@ -89,12 +89,6 @@ function checkByProductPriceType(elem) {
 
 }
 
-Event.observe(window, 'load', function () {
```

Comment:
Can you please move this code change to a separate PR as this is not part of a backport